brainwidemap/intro_figure_example_time_series.py

```diff
@@ -0,0 +1,334 @@
+from one.api import ONE
+import brainbox.behavior.wheel as wh
+from brainbox.processing import bincount2D
+from ibllib.atlas import AllenAtlas
+from ibllib.atlas.regions import BrainRegions
+from brainbox.io.one import SpikeSortingLoader
+from brainbox.io.one import SessionLoader
+import ibllib
+
+import numpy as np
+from pathlib import Path
+import math
+import matplotlib.pyplot as plt
+import pandas as pd
+import matplotlib as mpl
+
+
+
+mpl.rcParams.update({'font.size': 10})
+
+one = ONE(base_url='https://openalyx.internationalbrainlab.org',
+          password='international', silent=True)
+
+# save results for plotting here
+pth_res = Path(one.cache_dir, 'brain_wide_map', 'motor_correlates')
+pth_res.mkdir(parents=True, exist_ok=True)
+
+ba = AllenAtlas()
+br = BrainRegions()
+
+T_BIN = 0.02
+Fs = {'left': 60, 'right': 150, 'body': 30}
+
+
+blue_left = [0.13850039, 0.41331206, 0.74052025]
+red_right = [0.66080672, 0.21526712, 0.23069468]
+cdi = {0.8: blue_left, 0.2: red_right, 0.5: 'g', -1: 'cyan', 1: 'orange'}
+
+
+def get_allen_info():
+    '''
+    Function to load Allen region info, e.g. region colors palette
+    '''
+
+    p = (Path(ibllib.__file__).parent /
+         'atlas/allen_structure_tree.csv')
+
+    dfa = pd.read_csv(p)
```

Comment:
There is no function to load this in ibllib? Please double check

Reply:
I checked here and around the `ibllib.atlas` module:

https://int-brain-lab.github.io/iblenv/_modules/ibllib/atlas/regions.html#BrainRegions

It looks like the CSV is opened but the color hex values being manipulated in this code aren't exposed by the `BrainRegions` class. Doesn't look to me like there's a simple way to do it.